core/domain/email_manager.py

```diff
@@ -164,5 +181,60 @@ def send_post_signup_email(user_id):
         EMAIL_FOOTER.value)
 
     _send_email(
-        user_id, feconf.SYSTEM_COMMITTER_ID, email_models.INTENT_SIGNUP,
+        user_id, feconf.SYSTEM_COMMITTER_ID, feconf.EMAIL_INTENT_SIGNUP,
         email_subject, email_body)
+
+
+def require_valid_intent(intent):
+    if intent not in feconf.VALID_MODERATOR_ACTIONS:
+        raise Exception('Unrecognized email intent: %s' % intent)
+
+
+def _get_email_config(intent):
+    require_valid_intent(intent)
+    return config_domain.Registry.get_config_property(
```

Comment:
Good point, I changed the message. The help text in the editor UI should be sufficient.

However, I'm leaving the message where it is. These config options should only be edited in feconf or in the admin panel, but not both, and none of our other config properties declare their default values in feconf. In any case, this is just the default value for the config option, and app owners can set it to whatever they wish in the admin panel.